Source/Core/Core/NetPlayServer.cpp

```diff
@@ -674,8 +739,8 @@ void NetPlayServer::OnTraversalStateChanged()
 void NetPlayServer::SendChatMessage(const std::string& msg)
 {
 	auto spac = std::make_unique<sf::Packet>();
-	*spac << (MessageId)NP_MSG_CHAT_MESSAGE;
-	*spac << (PlayerId)0; // server id always 0
+	*spac << static_cast<MessageId>(NP_MSG_CHAT_MESSAGE);
```

Comment:
These changes are unrelated to the commit messages, and should be documented as such.